files/Aboot/boot0.j2

```diff
@@ -509,8 +565,8 @@ write_image_specific_cmdline() {
     fi
 
     # Obtain root partition uuid
-    local rootdev="$(mount | grep -E "$flash_re" | cut -f1 -d' ')"
-    local rootfstype="$(mount | grep -E "$flash_re" | cut -f5 -d' ')"
+    local rootdev="$(mount | grep "$target_path " | cut -f1 -d' ')"
```

Comment:
>mount [](start = 21, length = 5)

you can save `mount` stdout into a variable and reuse later.

Reply:
will store the mount line related to `target_path` related information in a local variable.